Local platform Keycloak Issue with RSA

Locally running the platform runs into keycloak issues:
```
hmda-platform akka.actor.ActorInitializationException: akka://hmda2/user/HmdaApi/hmda-filing-api: exception during creation
hmda-platform 	at akka.actor.ActorInitializationException$.apply(Actor.scala:196)
hmda-platform 	at akka.actor.ActorCell.create(ActorCell.scala:661)
hmda-platform 	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:513)
hmda-platform 	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:535)
hmda-platform 	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:295)
hmda-platform 	at akka.dispatch.Mailbox.run(Mailbox.scala:230)
hmda-platform 	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
hmda-platform 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
hmda-platform 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
hmda-platform 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
hmda-platform 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
hmda-platform 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
hmda-platform Caused by: java.security.spec.InvalidKeySpecException: java.security.InvalidKeyException: RSA keys must be at least 512 bits long
hmda-platform 	at java.base/sun.security.rsa.RSAKeyFactory.engineGeneratePublic(RSAKeyFactory.java:239)
hmda-platform 	at java.base/java.security.KeyFactory.generatePublic(KeyFactory.java:352)
hmda-platform 	at hmda.auth.KeycloakTokenVerifier.<init>(KeycloakTokenVerifier.scala:34)
hmda-platform 	at hmda.auth.OAuth2Authorization$.apply(OAuth2Authorization.scala:177)
hmda-platform 	at hmda.api.http.HmdaFilingApi$.$anonfun$apply$1(HmdaFilingApi.scala:40)
hmda-platform 	at akka.actor.typed.internal.BehaviorImpl$DeferredBehavior$$anon$1.apply(BehaviorImpl.scala:119)
hmda-platform 	at akka.actor.typed.Behavior$.start(Behavior.scala:168)
hmda-platform 	at akka.actor.typed.internal.adapter.ActorAdapter.preStart(ActorAdapter.scala:281)
hmda-platform 	at akka.actor.Actor.aroundPreStart(Actor.scala:548)
hmda-platform 	at akka.actor.Actor.aroundPreStart$(Actor.scala:548)
hmda-platform 	at akka.actor.typed.internal.adapter.ActorAdapter.aroundPreStart(ActorAdapter.scala:256)
hmda-platform 	at akka.actor.ActorCell.create(ActorCell.scala:641)
hmda-platform 	... 10 common frames omitted
hmda-platform Caused by: java.security.InvalidKeyException: RSA keys must be at least 512 bits long
hmda-platform 	at java.base/sun.security.rsa.RSAKeyFactory.checkKeyLengths(RSAKeyFactory.java:167)
hmda-platform 	at java.base/sun.security.rsa.RSAKeyFactory.checkRSAProviderKeyLengths(RSAKeyFactory.java:145)
hmda-platform 	at java.base/sun.security.rsa.RSAPublicKeyImpl.<init>(RSAPublicKeyImpl.java:92)
hmda-platform 	at java.base/sun.security.rsa.RSAKeyFactory.generatePublic(RSAKeyFactory.java:340)
hmda-platform 	at java.base/sun.security.rsa.RSAKeyFactory.engineGeneratePublic(RSAKeyFactory.java:235)
hmda-platform 	... 21 common frames omitted
```
pointing to `KeycloakTokenVerifier`

--- a/common/src/main/scala/hmda/auth/KeycloakTokenVerifier.scala
+++ b/common/src/main/scala/hmda/auth/KeycloakTokenVerifier.scala
@@ -31,7 +31,7 @@ class KeycloakTokenVerifier(keycloakDeployment: KeycloakDeployment) extends Toke
   val urlDecoder = Base64.getUrlDecoder
   val modulus = new BigInteger(1, urlDecoder.decode(config.getString("keycloak.publicKey.modulus")))
   val publicExponent = new BigInteger(1, urlDecoder.decode(config.getString("keycloak.publicKey.exponent")))
-  val publicKey = keyFactory.generatePublic(new RSAPublicKeySpec(modulus, publicExponent))
+  lazy val publicKey = keyFactory.generatePublic(new RSAPublicKeySpec(modulus, publicExponent))
 
   def verifyToken(token: String): Try[AccessToken] = {
     val tokenVerifier = TokenVerifier.create(token, classOf[AccessToken])

